packages/augur-state/src/db/AbstractDB.ts

```diff
@@ -14,9 +14,13 @@ export abstract class AbstractDB {
   protected db: PouchDB.Database;
   public readonly dbName: string;
 
-  constructor (dbName: string) {
+  protected constructor (dbName: string) {
     this.dbName = dbName;
-    this.db = new PouchDB(`db/${dbName}`);
+    this.db = this.makeDb();
```

Comment:
I put this into a separate function to make the test wrapper a little easier and more maintainable.